manifests/lcm_config.pp

```diff
@@ -0,0 +1,12 @@
+define dsc::lcm_config (
+  $refresh_mode = 'Disabled'
+) {
+
+  validate_re($refresh_mode, '^(Disabled|Pull|Push)$', 'refresh_mode must be one of \'Disabled\', \'Push\', \'Pull\'')
```

Comment:
Not to quibble, but the regex is case sensitive and Windows people are use to insensitive casing. May want to expect upper and lower case.